### PR TITLE
COMMERCE-505 Fixed orders back url

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:retest

**Comment by ethanbustad** [comment]
ci:stop

**Comment by ethanbustad** [comment]
Annnd apparently ci:stop doesn't work any more.....

File: commerce-order-web/src/main/resources/META-INF/resources/edit_order.jsp [hunk 1]
```diff
@@ -22,7 +22,13 @@ CommerceOrderEditDisplayContext commerceOrderEditDisplayContext = (CommerceOrder
 CommerceOrder commerceOrder = commerceOrderEditDisplayContext.getCommerceOrder();
 
 portletDisplay.setShowBackIcon(true);
-portletDisplay.setURLBack(String.valueOf(renderResponse.createRenderURL()));
+
+if (Validator.isNull(redirect)) {
+	portletDisplay.setURLBack(String.valueOf(renderResponse.createRenderURL()));
+}
+else {
+	portletDisplay.setURLBack(String.valueOf(redirect));
```
Comment (ethanbustad) [inline commerce-order-web/src/main/resources/META-INF/resources/edit_order.jsp]: @Commando18, isn't `redirect` already a `String`? We shouldn't have to call `String.valueOf` on it, right?

Comment (marco-leo) [reply commerce-order-web/src/main/resources/META-INF/resources/edit_order.jsp]: @Commando18 please fix as requested

**Comment by alessiorendina** [comment]
ci:retest

**Pull request merged**